test/e2e/Dockerfile.tests

```diff
@@ -0,0 +1,7 @@
+FROM fedora:33
+
+RUN INSTALL_PKGS="tcpdump frr iputils iproute nc hostname" && \
```

Comment:
this commit is not needed becaause you already added this dockerfile to a different location in the pod gw tests PR right?

Reply:
Right will remove it, must be a side effect of the rebase

Reply:
Removed